packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:math';
```

Comment:
This is usually imported "as math"; used with the prefix like `math.max`
```
import 'dart:math' as math;
```

Reply:
Done.